cpp/benchmarks/io/utilities/batched_memset_bench.cpp

```diff
@@ -1,101 +0,0 @@
-/*
```

Comment:
I wonder if it the empty `cpp/benchmarks/io/utilities` folder is also deleted in this PR.